bzt/jmx2yaml.py

```diff
@@ -1273,7 +1270,7 @@ def _get_global_objects(self):
         self.global_objects = []
         try:
             ht_object = self.tree.find(".//hashTree").find(".//TestPlan").getnext()
-        except:
+        except BaseException:
             raise TaurusInternalException("Bad jmx format")
```

Comment:
Is AttributeError wide enough?

Reply:
Not sure. I just changed this for PEP compliance.